tools/Azure.Mcp.Tools.Speech/src/Options/Stt/SttRecognizeOptions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace Azure.Mcp.Tools.Speech.Options.Stt;
+
+public class SttRecognizeOptions : BaseSpeechOptions
+{
+    [JsonPropertyName(SpeechOptionDefinitions.FileName)]
+    public string? File { get; set; }
```

Comment:
No need to follow-up on this PR, general question to @jongio 

I see this pattern intermittently where `Options` types have JSON property name usage and where they don't. Should we make a design guideline for this, and which choice should we use?

Reply:
I only want to see them if the name is different than the property name